pyfr/plugins/fluidforce.py

```
@@ -21,6 +22,19 @@ def __init__(self, intg, cfgsect, suffix):
         # Output frequency
         self.nsteps = self.cfg.getint(cfgsect, 'nsteps')
 
+        # Pressure only or viscous force
+        self.forcetype = self.cfg.get(cfgsect, 'type', 'viscous')
+        if self.forcetype not in {'pressure', 'viscous'}:
+            raise ValueError('Invalid fluid force type')
+        elif self.forcetype == 'viscous' and intg.system.name == 'euler':
+            self.forcetype = 'pressure'
```

I think we should raise an error here.